src/mercury_prof_interface.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2013-2020 Argonne National Laboratory, Department of Energy,
+ *                    UChicago Argonne, LLC and The HDF Group.
+ * All rights reserved.
+ *
+ * The full copyright notice, including terms governing use, modification,
+ * and redistribution, is contained in the COPYING file that can be
+ * found at the root of the source code distribution tree.
+ */
+
+#ifndef MERCURY_PROF_INTERFACE_H
+#define MERCURY_PROF_INTERFACE_H
+
+#include "mercury_prof_types.h"
+
+/*************************************/
+/* Public Type and Struct Definition */
+/*************************************/
+
+/* See mercury_prof_types.h */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Client-side API to initialize the PVAR profiling interface.
+ *
+ * \return HG_SUCCESS or corresponding HG error code
+ */
+HG_PUBLIC hg_return_t 
+HG_Prof_init();
+
```

Comment:
That should probably take an hg_class as a parameter. We usually try to avoid using globals.

Reply:
See my previous comment, you should return an `hg_prof_class_t *` here